fairscale/utils/parallel.py

```diff
@@ -80,6 +90,10 @@ def get_process_group_cached(ranks: Optional[Sequence[int]] = None) -> ProcessGr
     Extra process groups can also reduce training speed (observed on VISSL models).
 
     Args:
+        name ProcessGroupName:
+            There are two process groups when reduce_scatter overlap is enabled. The "default" process group is the
```

Comment:
what about the "all_gather" process group in the enum list above?

Reply:
it is removed.